md/performance-tuning.md

```diff
@@ -487,3 +485,63 @@ Details on these properties can be found in [the Quartz documentation](http://ww
 
 They are not read subsequently, so changing the values in `bonita-tenant-community-custom.properties` after the Engine has been started has no effect on Quartz.
 For value definition, and information about how to update the Quartz trigger tables, see the [Quartz documentation](http://www.quartz-scheduler.org/documentation/) about Cron Triggers.
+
+
+#### PostgreSQL performance tuning
+
+Here is Bonita advice to finely tune PostgreSQL database server performance.
+
+In this example, we assume with have:
+* 12Gb of RAM
+* fast SSD storage
+
+Update **memory** configuration in file `postgresql.conf` (typically `/etc/postgresql/11/main/postgresql.conf`) with the
+following values:
+
+```properties
+# MEMORY PARAMETERS:
+# shared_buffers SHOULD be set to 1/4 of the total memory available on the server, with a maximum of 8GB:
+shared_buffers = 3GB
+work_mem = 16MB
+maintenance_work_mem = 256MB
+
+# QUERY PLANNING PARAMETERS:
+# cost of non-sequentially-fetched disk page. 2 for fast RAID0 disks, higher value for slower disks:
+random_page_cost = 2
+# cost of a disk page fetch. Value is correlated with random_page_cost. See Warning below. :
+seq_page_cost = 2
+# effective_cache_size SHOULD be 2/3 of the total memory available on the server
+effective_cache_size = 8GB
+# effective_io_concurrency is the number of current disk operations. 200 is a good value for SSD.
+effective_io_concurrency = 200
+checkpoint_completion_target = 0.9
+```
+
+::: warning
+Warning: properties `random_page_cost` and `seq_page_cost` should have values relative to each other thoroughly set, in order
+for PostgreSQL query planner to choose the right execution plan.  
+See PostgreSQL [Planner Cost Constants](https://www.postgresql.org/docs/11/runtime-config-query.html#RUNTIME-CONFIG-QUERY-CONSTANTS)
+for more details on how to set those values.
+:::
+
+If you want to be able to **restore live PITR backup** of the database, ensure archiving is activated:
```

Comment:
We could tell what is `PITR` or at least only write in parenthesis the meaning of the acronym)